cpp/CMakeLists.txt

```diff
@@ -275,14 +283,24 @@ set(CUOPT_PRIVATE_CUDA_LIBS
   CUDA::curand
   CUDA::cusolver
   TBB::tbb
+  CUDA::cuda_driver
   OpenMP::OpenMP_CXX)
 
 list(PREPEND CUOPT_PRIVATE_CUDA_LIBS CUDA::cublasLt)
 
+
+set_property(TARGET cuopt APPEND PROPERTY
+  RAPIDS_LINK_LIBS_EXCLUSIONS "libcuda.so.1"
+)
```

Comment:
Is this necessary? If you're using driver APIs you need to link to the driver, right?